.travis.yml

```diff
@@ -60,10 +60,14 @@ matrix:
         apt:
           sources:
             - ubuntu-toolchain-r-test     # For gcc 4.9, 5 and 7
+            - llvm-toolchain-xenial-6.0
           packages:
             - gcc-7
             - g++-7
-      env: DEVITO_ARCH=gcc-7 DEVITO_OPENMP=0 DEVITO_BACKEND=ops YC_CXX=g++-7 INSTALL_TYPE=conda RUN_EXAMPLES=False MPI_INSTALL=0
+            - wget
+            - unzip
+            - tree
+      env: DEVITO_ARCH=gcc-7 DEVITO_OPENMP=0 DEVITO_BACKEND=ops YC_CXX=g++-7 INSTALL_TYPE=conda RUN_EXAMPLES=False MPI_INSTALL=0 CUDA_PATH=/usr/local/cuda CUDA_BIN_PATH=/usr/local/cuda/bin
```

Comment:
Note for myself, this is starting to be way to many env variable there, same for other build, need to rethink it. Part of CI refactoring

Reply:
it does indeed